migrations/Version20220310155818.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20220310155818 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('CREATE UNIQUE INDEX UNIQ_2D4B9BCED17F51A8 ON file (uuid);');
+        $this->addSql('ALTER TABLE file DROP extension;');
+        $this->addSql('ALTER TABLE file DROP type;');
+        $this->addSql('ALTER TABLE file DROP mime_type;');
+        $this->addSql('ALTER TABLE file DROP size;');
+        $this->addSql('ALTER TABLE file ADD path VARCHAR(255) NOT NULL;');
+    }
```

Comment:
Tu peux mettre les drops à la suite des autres comme ça :
```suggestion
        $this->addSql('ALTER TABLE file DROP extension, DROP type, DROP mime_type, DROP size, ADD path VARCHAR(255) NOT NULL;');
    }
```

C'est possible que ce soit mal écris, il faudra peut-être testé